api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -13,7 +13,7 @@
 ALLOWED_SUBMITTERS = {"scpca"}
 # NOTE: When INPUT_BUCKET_NAME is changed, please delete the contents of
 # api/test_data/input before testing to ensure test files are updated correctly.
-INPUT_BUCKET_NAME = "scpca-portal-public-test-inputs/2024-04-19/"
+INPUT_BUCKET_NAME = common.INPUT_BUCKET_NAME
```

Comment:
Let's do the following:
- update comment above so it captures the following:
  - test data bucket is defined in common.py
  - when that value changes be sure to wipe your local folder to ensure files are updated
- remove the local `INPUT_BUCKET_NAME` constant
- clean up where we explicitly pass `INPUT_BUCKET_NAME` into `load_data` in this file
- in `load_data` (management command file) replace hard-coded `scpca-portal-inputs` with `common.INPUT_BUCKET_NAME`